estimation/alternatives.R

```diff
@@ -199,12 +199,34 @@ for (i in rows.along(input)) {
     columns = read.delims(fname)
     columns$column = sprintf("^%s$", columns$column)
     
-    data_columns = write_estimation_data(alternatives=alternatives,
-                                         batch_size=100,
-                                         model_name=input$name[i],
-                                         row=row,
-                                         matrix_list=matrix_list,
-                                         columns=columns)
+    if (input$name[i] == "metropolitan") {
+        data_columns = write_estimation_data(alternatives=subset(alternatives, ttype %in% c(1,2,3)),
```

Comment:
I assume this filters out the relevant ttypes? If yes, it would improve readability and maintainability if we'd have named constants instead of 1,2&3... 

```
ttype %in% c(1,2,3))
```

Reply:
6acb6f84535dbf49d1e486721d0847a2eeab5879 Not a perfect fix, but improves readability. `ttype` variable goes into Alogit software and can not be anything else than a numeric.